include/deal.II/dofs/dof_handler.h

```diff
@@ -256,7 +256,35 @@ class DoFHandler  :  public Subscriptor
    */
   typedef typename ActiveSelector::cell_iterator        cell_iterator;
 
+  /**
+   * A typedef that is used to identify iterators that point to faces.
+   * The concept of iterators is discussed at length in the
+   * @ref Iterators "iterators documentation module".
+   *
+   * While the actual data type of the typedef is hidden behind a few layers
+   * of (unfortunately necessary) indirections, it is in essence
+   * TriaIterator<DoFAccessor>. The
+   * TriaIterator class works like a pointer to objects that when
+   * you dereference it yields an object of type DoFAccessor. DoFAccessor,
+   * in turn, is a class that can be used to query DoF indices on faces,
+   * but it is also derived from TriaAccessor and consequently be used
+   * to query geometric properties such as their vertices, their area, etc.
```

Comment:
maybe `..such as vertices of faces, their area...` as readers probably already forgotten what `their` is.